getonlineplayers() is a collection now. It can not be converted to players[ ]

Sycholic opened the pull request:
Fixed and verified this with some people in spigot's irc they agreed this would be the correct way now.  Let me know what you think.

Commit:
getonlineplayers() is a collection now. It can not be converted to players[ ]

vk2gpz commented:
Thanks for letting me know. They should not do this sort to the public methods...

Sycholic commented:
I dunno if or when it was changed I just happened to notice the big red flag and netbeans screaming at me that its a collection.  so I worked out what would be the equal to what the original was doing.

Commit:
Fixed 2 lists that had a oddball ' , ' at the end of the list Im hoping this was the NPE source.

Commit:
Added Maven animal snipper for java version control

Commit:
excluded/ignored

Commit:
ignored from commit for one reason, we dont include spigot/CB jars...

Sycholic committed:
Delete spigot-1.8-R0.1-SNAPSHOT.jar

because this crap cant be in here.....

Samistine committed:
Get Bukkit from Spigot Repo

Commit:
Recommitting for Jenkins

Samistine committed:
We are getting Bukkit from Spigot's repo

Samistine committed:
Fix Grammar

Commit:
Added animal-sniffer for maven

Keeps the bytecode 1.7 compatible. there are differences in Java7 and 8.
Dont wanna hear someone crying we compiled on 8 and dont work because
they trying to run in on Java7 ^_^

Commit:
Formatting Cleanup

Sycholic commented:
dafuk why is it keep on adding stuff I have not asked to get pulled... welp guess Im closing this PR.
if you want to add in the 
20fc6d4 and d2ec717 those are the only things that should be pulled if anything I never told github to PR anything infact other then 20fc6d4... GG github
